infrastructure/src/test/java/org/wso2/testgrid/infrastructure/AWSProviderTest.java

```diff
@@ -80,6 +84,27 @@
     private String scriptFile = "template.json";
     private String mockStackName = "MockStack";
 
+    private TestPlan testPlan;
+    private List<Script> scripts;
+    private String scenarioRepo = "src/test/resources/scenarios";
```

Comment:
Shall we use java.nio.Paths API here to define path platform independently? Also better to have scenarioRepo as a static final variable.

Reply:
Made changes in e4af8be